config.json

```diff
@@ -1,4 +1,4 @@
 {
-	"token": "MTA1NDAzNjUwNDI5ODY1MTc1OQ.GoFiqc.eTw2btMw8AiYxHCX_9VqguXLouUI2q9cG_eXW0",
+	"token": "",
```

Comment:
next time place your config file in gitignore, or dont commit it. its fine for now. i made this mistake myself way too often too